test/integration/goldens/kms/src/com/google/cloud/kms/v1/KeyManagementServiceClient.java

```diff
@@ -3685,9 +3685,7 @@ public final UnaryCallable<GetIamPolicyRequest, Policy> getIamPolicyCallable() {
 
   // AUTO-GENERATED DOCUMENTATION AND METHOD.
   /**
-   * Lists information about the supported locations for this service.
-   *
-   * <p>Sample code:
+   * Sample code:
```

Comment:
This is the last file with comment issue.

Reply:
After some investigation, I think this is because `LocationsProto.getDescriptor()` doesn't contain `sourceCodeInfo_`

Reply:
Now this comment is restored.

Reply:
What was the fix?

Reply:
If the protos are dependent on mixins or mixins are passed through command line, then we don't add mixin file descriptors to `fileDescriptors` because the latter don't have SourceCodeInfo.

https://github.com/googleapis/sdk-platform-java/blob/ab6f4944df4ffc6a650a1a1e53d7302a64524933/gapic-generator-java/src/main/java/com/google/api/generator/gapic/protoparser/Parser.java#L1062-L1063

Then use file descriptor in `fileDescriptors`, rather than `MIXIN_ALLOWLIST`.

https://github.com/googleapis/sdk-platform-java/blob/ab6f4944df4ffc6a650a1a1e53d7302a64524933/gapic-generator-java/src/main/java/com/google/api/generator/gapic/protoparser/Parser.java#L275

Though, in the future, if the protos are not dependent on mixins and no mixins are provided, the mixin method description (not the sample code) might lose. The main issue is I don't know how to get `SourceCodeInfo` from `getFileDescriptor` method.